api/catalog/api/utils/validate_images.py

```diff
@@ -99,7 +112,7 @@ def validate_images(query_hash, start_slice, results, image_urls):
     for idx, url in enumerate(to_verify):
         cache_idx = to_verify[url]
         if verified[idx] is not None:
```

Comment:
I might be wrong, won't this always be true? I believe the type of `verified[idx]` is `tuple[str, int]` which won't be `None` even in the default case.

I think this will still work because the default is now `-1`, but that also means the `else` code path won't be touched.

Reply:
I think you're right. I'll make the clean up here, thanks for catching it!